### DISCUSSION TITLE
XAML for UI design

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [ppy/osu-framework#5244] [Ideas] [answered]
It would be nice if we can employ existing XAML knowledge from Xamarin/MAUI to draw UI elements, especially that MVVM design is suitable for dynamic environment

**Comment by bdach** [comment, marked as answer]
related: https://github.com/ppy/osu-framework/issues/3056

that said i'm not a huge fan of the XAML way. it tends to get pretty verbose for no real good reason.
